### PR TITLE
Add new default avatar

### PR DESCRIPTION
**Pull request by jnordberg**

### CODE REVIEW

File: imagehoster/app/server/data-server.js [hunk 1]
```diff
@@ -18,14 +18,14 @@ const {uploadBucket} = config
 const router = require('koa-router')()
 
 const assetRoot = path.resolve(__dirname, '../..');
-router.get('/u/__default/avatar', function* () {
+router.get('/u/__default_new/avatar', function* () {
```
Comment (jnordberg) [inline imagehoster/app/server/data-server.js]: Url changed so we don't have to delete the old avatar from the bucket

Comment (relativityboy) [reply imagehoster/app/server/data-server.js]: Our next avatar will be `_default_new_(2)` ...

Comment (jnordberg) [reply imagehoster/app/server/data-server.js]: The reason for this is that the image proxy never expires proxied urls. Not going to be a problem when the rewrite is done

**Comment by sneak** [comment]
@relativityboy beat me

**Pull request merged**